site/content/entry/yuzu-progress-report-nov-2020/index.md

```diff
@@ -0,0 +1,169 @@
+
+{{< imgs
+	"./botw.mp4| Prettiest fast-traveling (The Legend of Zelda: Breath of the Wild)"
+  >}}
+
+One sad point of this rewrite is that the “Vulkan memory manager”, now called the `Texture Reaper`, which was expected to help reduce VRAM use in Vulkan, will take longer to implement due to technical complications that surfaced during its development. There is more detailed information in the dedicated article.
+
+A small PSA, AMD users should install the `20.11.3` driver version or newer, as it adds `VK_EXT_robustness2` to all missing Radeon products, helping in stability and precision.
+
+## Project Aether, or how to visit yuzu from within yuzu
+
+Nintendo Switch games have access to a `web applet`, used to show tutorials or guides while playing. Our original implementation using [QtWebEngine](https://doc.qt.io/qt-5/qtwebengine-index.html) had some lasting bugs that impeded gameplay in some cases, like the tutorials on a first boot of Super Smash Bros. Ultimate.
+
+[Morph](https://github.com/Morph1984) [rewrote yuzu’s web applet](https://github.com/yuzu-emu/yuzu/pull/5042), still based on QtWebEngine, but completely overhauled, now accepting control input instead of only touch emulation via mouse, and rendering native fonts extracted from the native OS on the Switch, or using Open Source Software fallbacks.
+
+{{< imgs
+	"./webapplet.png| Working tutorials in Super Smash Bros. Ultimate"
+  >}}
+  
+Previously non-rendering applets are now working, this includes the few cases where games got stuck due to the previous implementation, like in `Super Smash Bros. Ultimate`. Mods that bypass the applet are no longer required, like in `Super Mario Odyssey`.
+
+There are limitations listed in the Pull Request that we will continue to work on. Expect this change to force our minimum Ubuntu version requirement to 20.10 in the future due to the need to update Qt to version 5.14.2 or higher.
+
+{{< imgs
+	"./yuzu.png| yuzuception!"
+  >}}
+
+## General bug fixes and improvements
+
+[Rodrigo](https://github.com/ReinUsesLisp) brings the Xenoblade fans a nice little fix. [Implementing `early fragment tests`](https://github.com/yuzu-emu/yuzu/pull/5013) fixes ghost geometry problems that resulted in dark rendering areas.
+
+{{< single-title-imgs
+    "Like day and night! (Xenoblade Chronicles Definitive Edition)"
+    "./xcdebug.png"
+    "./xcdefix.png"
+  >}}
+
+This fix will be a Vulkan exclusive for now, to avoid invalidating the current OpenGL shader cache.
+
+More vulkan fixes! Now by [epicboy.](https://github.com/ameerj) Our resident fish [implemented the missing `alpha test culling` feature](https://github.com/yuzu-emu/yuzu/pull/4946) from Vulkan, removing transparency related rendering bugs. This could be easily spotted in the main menu of `Super Smash Bros. Ultimate` or the vegetation of `Tales of Vesperia: Definitive Edition`.
```

Comment:
`Our resident fish` doesn't make any sense to new readers or those unfamiliar with his discord profile picture (not even using the same photo on github). I'd remove it and substitute `He`